.github/workflows/dart_frog_auth.yaml

```diff
@@ -0,0 +1,28 @@
+name: dart_frog_auth
```

Comment:
We will also need a workflow for the example and dependabot updates for the package and the example.

Reply:
@renancaraujo I am not sure I know how to set that up, can you advise me on this?

Reply:
Yep, for the example app, you can use the existing examples as inspiration. For example (no pun intended) see [the workflow the todo](https://github.com/VeryGoodOpenSource/dart_frog/blob/main/.github/workflows/examples_todos.yaml).

For the package, see the workflow for [the WebSocket one](https://github.com/VeryGoodOpenSource/dart_frog/blob/main/.github/workflows/dart_frog_web_socket.yaml).

Reply:
Hm, I am confused, these files that you sent are the "normal" workflows right? The auth package already have one in this PR. The example app will be done in a followup PR.

I wonder about the dependabot updates that you mentioned